src/components/Scheme/SchemeContainer.tsx

```diff
@@ -53,6 +55,28 @@ const mapDispatchToProps = {
 
 class SchemeContainer extends React.Component<IProps, null> {
 
+  private getSchemeIsActive(scheme: ISchemeState): boolean {
```

Comment:
seems like this code should be in the client

Reply:
I agree, but we can first write it here and later port it to the client